queries.sql

```diff
@@ -74,3 +74,78 @@ SELECT COUNT(escape_attempts) AS escape_attempts, neutered FROM animals GROUP BY
 SELECT MAX(weight_kg), MIN(weight_kg), species FROM animals GROUP BY species;
 
 SELECT AVG(escape_attempts) AS escape_attempts, species FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- Modify your inserted animals so it includes the species_id value:
+UPDATE animals 
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals 
+SET species_id = 1
+WHERE species_id IS NULL;
+
+-- Modify your inserted animals to include owner information (owner_id):
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name = 'Devimon' OR name = 'Plantmon';
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name IN ('Angemon', 'Boarmon');
```

Comment:
- [ ] These `UPDATE` queries should be located inside data.sql on inside queries.sql. The reason because with these queries we are modifying the date.

More info:

> It is best to update query statements within the data.sql file because this will ensure that all of your queries are up to date and that any changes in your data are applied directly to your queries. Placing the updates inside the queries.sql file would make it less efficient to keep track of changes as they occur over time.